src/lib/misc.c

```diff
@@ -1139,4 +1140,76 @@ rnp_filename(const char *path)
     } else {
         return res + 1;
     }
-}
\ No newline at end of file
+}
+
+/* find the time - in a specific %Y-%m-%d format - using a regexp */
+int
+grabdate(const char *s, int64_t *t)
```

Comment:
This function was just moved 'as it is' from the write.c or something similar.
I'm chasing 1 to RNP_OK, and 0 to RNP_FAIL, and logically those return values are ints. While here it is bool. But we don't use bool anywhere so it is int.